generators/typescript/sdk/generator/src/TemplateGenerator.ts

```diff
@@ -488,14 +498,16 @@ export class TemplateGenerator {
         location,
         wireOrOriginalName,
         nameBreadcrumbs,
-        indentationLevel
+        indentationLevel,
+        isUnionType
     }: {
         typeReference: TypeReference;
         name: string | undefined;
         location: FdrSnippetTemplate.PayloadLocation;
         wireOrOriginalName: string | undefined;
         nameBreadcrumbs: string[] | undefined;
         indentationLevel: number;
+        isUnionType?: boolean;
```

Comment:
Nit: mind making this `| undefined` as opposed to `?`? Just makes it harder to miss passing it in!

Reply:
Made this required instead!